package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lev-api",
-  "version": "0.16.1",
+  "version": "0.16.2",
```

Comment:
You'll also need to make a similar change to `.drone.yml`

Reply:
All done